.buildkite/pipelines/es_serverless/verify_es_serverless_image.yml

```
@@ -19,6 +19,12 @@ steps:
     command: |
       buildkite-agent annotate --context es-serverless-image --style info "ES Serverless image: $ES_SERVERLESS_IMAGE"
       buildkite-agent annotate --context kibana-commit --style info "Kibana build hash: $BUILDKITE_BRANCH / $BUILDKITE_COMMIT"
+      cat << EOF | buildkite-agent annotate --context es-serverless-run --style info
+        To run this locally:
+        \`\`\`
+        node scripts/es serverless --image docker.elastic.co/elasticsearch-ci/elasticsearch-serverless:$ES_SERVERLESS_IMAGE
```

We can also just use the tag if you prefer shorter

```suggestion
        node scripts/es serverless --tag $ES_SERVERLESS_IMAGE
```

---

Thanks, I pushed 2f70526ce1b5963efda7970ee5b61fab158867f9